doc/authentication-methods/client-certificate.md

```diff
@@ -46,11 +46,33 @@ Below you may find a list of backends that are safe to use with `cyrsasl_externa
 * `rdbms`
 * `riak`
 
+### Self-signed certificates
+
+By default MongooseIM doesn't accept self-signed certs for the SASL-EXTERNAL authentication.
+They may be handy in development so we made it possible to tell MongooseIM to accept them.
+
+#### Self-signed certificates for regular TCP/TLS connections
+
+In order to tell MongooseIM to accept self-signed certs, `ssl_options` list needs to be added to `ejabberd_c2s` listener config like below:
+
+```Erlang
+{ssl_options, [{verify_fun, {selfsigned_peer, true}}]}
```

Comment:
Actually this fast_tls thing is so under-documented (and tested) that it deserves it's own PR. (already noted that in Jira).
Do you still want me to change the doc or can it be a subject of another PR?

Reply:
I still think it should be documented in this PR because we introduce new config value with some mysterious flag. We don't need to dive into implementation details, simple explanation about "disconnect on verification failure" would be sufficient.

Reply:
The truth is the `ssl_optinos` was introduce along with `fast_tls`. I wonder why a PR with so under-documented and under-tested module was accepted... OK, I'll add a short description of the `true` value.